src/upgrade.js

```diff
@@ -4,6 +4,8 @@ const chalk = require('chalk');
 const semver = require('semver');
 const execa = require('execa');
 const fetch = require('node-fetch');
+const { mostRecentVersion, toVersion } = require('fetchUpdates');
```

Comment:
fetchUpdates returns (but doesn't export) `mostRecentVersion` and `newestVersion`. This file tries to import `mostRecentVersion` and `toVersion`. I think you might mean `newestVersion` instead of `toVersion`

Reply:
You are correct sir